tools/mo/unit_tests/mo/moc_frontend/moc_extractor_test_actual.py

```diff
@@ -110,23 +109,23 @@ def test_decode_name_with_port_delim_tensor_no_collision_out(self):
         model_stat = get_model_statistic()
         place_stat = get_place_statistic()
 
-        assert model_stat.get_place_by_tensor_name == 1
+        assert model_stat.get_place_by_tensor_name > 1
```

Comment:
In case if someone in future makes changes and avoids redundant call - at lest there will be no need to edit this test back to == 1
```suggestion
        assert model_stat.get_place_by_tensor_name >= 1
```